source/ui/Components/Output/qml.qrc

```diff
@@ -4,6 +4,7 @@
         <file>RedRectangle.qml</file>
         <file>GreenRectangle.qml</file>
         <file>BlueRectangle.qml</file>
+        <file>PixelDisplay.qml</file>
```

Comment:
I think this file is not used. Use the `source/ui/qml.qrc` instead. This file should be deleted.